src/sandbox/jakub/index.tsx

```diff
@@ -1,9 +1,12 @@
 import React from 'react'
 
+import LoginPage from './login'
+
 const JakubSandboxPage: React.FC = () => {
   return (
     <>
       <h1>Jakub Sandbox</h1>
+      <LoginPage></LoginPage>
```

Comment:
Jeżeli nie przekazujemy żadnego `children` wewnątrz komponentu to powinniśmy go od razu zamykać
```suggestion
      <LoginPage/>
```